### PR TITLE
chore: bump hermes binary to v1.10.0

### PR DESCRIPTION
**Pull request by Pantani**
## Description

- Bump Hermes binary to v1.10.0;
- Improve configure logs;
- Set the `channel-version` flag for the hermes create channel command;

### CODE REVIEW

**Review by julienrbrt — APPROVED** [review]
utACK!

**Comment by julienrbrt** [comment]
Can we add a changelog entry?

**Review by julienrbrt — DISMISSED** [review]
lgtm!

**Comment by julienrbrt** [comment]
The unit tests are failing: https://github.com/ignite/apps/actions/runs/9749828095/job/26907870139?pr=114

File: hermes/main.go [hunk 1]
```diff
@@ -16,18 +16,16 @@ type app struct{}
 
 func (app) Manifest(context.Context) (*plugin.Manifest, error) {
 	m := &plugin.Manifest{Name: "hermes"}
-	m.ImportCobraCommand(cmd.NewRelayer(), "ignite")
+	m.ImportCobraCommand(cmd.NewHermes(), "ignite relayer")
```
Comment (julienrbrt) [inline hermes/main.go:19]: Won't this require changes on cli main again?

**Review by julienrbrt — APPROVED** [review]

File: hermes/main.go (hunk 1 above)
Comment (Pantani) [reply hermes/main.go:19]: Yes, I reverted because we need to launch a CLI version without the builtin relayer

Comment (julienrbrt) [reply hermes/main.go:19]: Yeah that makes sense. I think it's better like this as the app will stay compatible with v29 and v28. We should avoid breaking changes so it's great 🙌

**Pull request merged**